src/Refactoring-Transformations/OCSequenceNode.extension.st

```diff
@@ -1,9 +1 @@
-Extension { #name : 'OCSequenceNode' }
-
-{ #category : '*Refactoring-Transformations' }
-OCSequenceNode >> addNodeLast: aNode [
-
-	self lastIsReturn
-		ifTrue: [ self addNode: aNode before: self statements last ]
-		ifFalse: [ self addNode: aNode ]
-]
+Extension { #name : 'OCSequenceNode' }
```

Comment:
This is weird that this file is still here if it is empty..

Did you just commit normally using Iceberg?